packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -378,12 +385,25 @@ export interface PluginModel {
         frontend?: string;
         backend?: string;
     };
+    packagePath: string;
+
+    // frontend contributions info -- start
     contributes?: PluginContribution;
+    activationEvents?: string[];
+    // frontend contributions info -- end
+
+    // backend dependencies info -- start
     /**
-     * The deployable form of extensionDependencies from package.json,
-     * i.e. not `publisher.name`, but `vscode:extension/publisher.name`.
+     * The deployable form of `rawExtensionDependencies`,
+     * e.g. not `publisher.name`, but `vscode:extension/publisher.name`.
      */
     extensionDependencies?: string[];
```

Comment:
also rawExtensionDependencies / extensionDependencies (extension) is VS Code name. It looks odd inside plug-in model to have extensions which may let thing it could be Theia extensions

Reply:
so it should be `extensionDependencies` and `rawPluginDependencies`? or we should break nor? this comment is probably very related to the scope of this PR

Reply:
yes, it's something that I've missed long time ago

Reply:
renamed